apisix/plugin.lua

```diff
@@ -371,6 +375,28 @@ local function trace_plugins_info_for_debug(ctx, plugins)
     end
 end
 
+local function meta_filter(plugin_name, plugin_conf)
+    local filter_vars = plugin_conf._meta and
+                plugin_conf._meta.filter and plugin_conf._meta.filter.vars
+    if not filter_vars then
+        return true
+    end
+
+    local ex, ok, err
+    ex, err = expr_lrucache(plugin_name, plugin_conf, expr.new, filter_vars)
```

Comment:
We can't use plugin_name as a key unless there is only one filter for a plugin.

Reply:
What I plan is only one filter for a plugin.Is this reasonable？

Reply:
However, we need one filter for each plugin configuration (route/x -> jwt-auth), not a particular plugin like `jwt-auth`